myuw/templates/handlebars/banner/profile.html

```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
 
             {{#with password}}
             <div class="profile-passwords row">
-                <h3 class="sr-only">Password Information</h3>
-                {{#if has_active_med_pw}}
-                <div class="col-md-6 uwmedicine-password">
-                    <h4>UW Medicine Account</h4>
-                    {{#if med_pw_expired }}
-                        <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
-                    {{ else }}
-                        <span class="password-status {{#if expires_in_30_days_or_less}}password-alert{{/if}}">Password expires in {{days_before_med_pw_expires}} days ({{formatDateAsDate expires_med}})</span><br>
-                        <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change UW Medicine password</a>
+                <div style="margin: 0 5px;">
+                    <h3 class="sr-only">Password Information</h3>
+                    {{#if has_active_med_pw}}
+                    <div class="col-md-6 uwmedicine-password">
+                        <h4>UW Medicine Account</h4>
+                        {{#if med_pw_expired }}
+                            <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
+                        {{ else }}
+                            <span class="password-status {{#if expires_in_30_days_or_less}}password-alert{{/if}}">Password expires in {{days_before_med_pw_expires}} days ({{formatDateAsDate expires_med}})</span><br>
+                            <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change UW Medicine password</a>
+                        {{/if}}
+                            <p>This password is for UW Medicine applications like Epic, ORCA, MINDscape, AMC network, etc.</p>
+                    </div>
+                    <div class="uwnetid-password col-md-6">
+                        <h4>UW NetID Account</h4>
+                        <span class="password-status">Password last updated {{days_after_last_pw_change}} days ago</span></br>
```

Comment:
You have a `</br>` at the end of this line. This should be either `<br>` or `<br />` if you want to be picky.